public/index.html

```diff
@@ -7,6 +7,9 @@
 
     <!-- Bootstrap CSS -->
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css" integrity="sha384-rwoIResjU2yc3z8GV/NPeZWAv56rSmLldC3R/AZzGRnGxQQKnKkoFVhFQhNUwEyJ" crossorigin="anonymous">
+		<link rel="stylesheet" href="style.css">
```

Comment:
Make sure indentation is preserved. Previous line has 4-spaces, this one should match.
I'll add a linter, so that you get feedback inside your code editors, but for now let's just roll with what's there.